File: logical.py
class Logical:
    def div(self,no):
        di = 2
        count = 0

        while (no > di):
            if(no % di == 0):
                print(di)
                count +=1
            di += 1

        print("count"+str(count))
        if count == 0:
            print("Prime Number")
        else:
            print("Not Prime")
            
div = Logical()
number = int(input("Enter Number"))
div.div(number)

File: logical.java
class Logic
{
public static void main(String[] args){
    Logic loop = new Logic();
    loop.diviser(131);
}
void diviser(int no)
{
    int div = 2;
    int count = 0;
    while(div < no)
    {
        if(no % div == 0)
        {
            System.out.println(div);
            count = count + 1;
        } 
    div += 1;   
    }
System.out.println("count : "+count);
if(count == 0)
    {
        System.out.println("Prime");  
    }
else
    {
        System.out.println("Not Prime");    
    }
}

}
